llvm/lib/Transforms/Instrumentation/LowerAllowCheckPass.cpp

```diff
@@ -182,11 +182,13 @@ void LowerAllowCheckPass::printPipeline(
   // correctness.
   // TODO: print shorter output by combining adjacent runs, etc.
   int i = 0;
+  bool printed = false;
```

Comment:
OS << llvm:::join(llvm::map_range())

Reply:
This is not the same? This is a vector with some elements 0, and we want to skip those

Reply:
Just an idea, not a request to change